### PR TITLE
chore: Migrate CI to github actions

### PR DESCRIPTION
**Pull request by sawood14012**
# Description

Enables Use of github actions to run integration tests.

- [X] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: .github/workflows/ci-run-e2e.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+              -e F8A_LICENSE_SERVICE_URL=${F8A_LICENSE_SERVICE_URL} \
+              -e MANIFESTS_BUCKET_URL=${MANIFESTS_BUCKET_URL} \
+              ${RECOMMENDER_API_TOKEN:+-e RECOMMENDER_API_TOKEN=${RECOMMENDER_API_TOKEN}} \
+              ${RECOMMENDER_REFRESH_TOKEN:+-e RECOMMENDER_REFRESH_TOKEN=${RECOMMENDER_REFRESH_TOKEN}} \
+              ${OSIO_AUTH_SERVICE:+-e OSIO_AUTH_SERVICE=${OSIO_AUTH_SERVICE}} \
+              ${THREE_SCALE_PREVIEW_USER_KEY:+-e THREE_SCALE_PREVIEW_USER_KEY=${THREE_SCALE_PREVIEW_USER_KEY}} \
+              ${REGISTERED_USER_UUID:+-e REGISTERED_USER_UUID=${REGISTERED_USER_UUID}} \
+              ${SNYK_TOKEN:+-e SNYK_TOKEN=${SNYK_TOKEN}} \
+              f8a-e2e-tests --tags=-jobs.requires_auth --tags=-requires.openshift.console.access --no-color $@
+
```
Comment (arajkumar) [inline .github/workflows/ci-run-e2e.yml:59]: Can you add comments to denote e2e execution is in progress? similar to what you have done for build-image.

File: .github/workflows/ci-run-e2e.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            F8A_3SCALE_URL=${F8A_3SCALE_URL:-https://3scale-connect.api.prod-preview.openshift.io/};
+            F8A_THREE_SCALE_PREVIEW_URL=${F8A_THREE_SCALE_PREVIEW_URL:-https://f8a-analytics-preview-2445582058137.production.gw.apicast.io};
+            F8A_BACKBONE_API_URL=${F8A_BACKBONE_API_URL:-http://f8a-server-backbone-bayesian-preview.b6ff.rh-idev.openshiftapps.com/};
+            F8A_SERVICE_ID=${F8A_SERVICE_ID:-2555417755633};
+            F8A_GEMINI_API_URL=${F8A_GEMINI_API_URL:-https://gemini.api.prod-preview.openshift.io/};
+            F8A_LICENSE_SERVICE_URL=${F8A_LICENSE_SERVICE_URL:-https://license-analysis.api.prod-preview.openshift.io};
+            OSIO_AUTH_SERVICE=${OSIO_AUTH_SERVICE:-https://auth.prod-preview.openshift.io};
+            MANIFESTS_BUCKET_URL=${MANIFESTS_BUCKET_URL:-https://public-dynamic-manifests.s3.amazonaws.com/};
+        - name: Build Docker Image 
+          run: docker build -t f8a-e2e-tests .
```
Comment (arajkumar) [inline .github/workflows/ci-run-e2e.yml:37]: Do we really need a docker image to run e2e?

File: .github/workflows/ci-run-e2e.yml (hunk 1 above)
Comment (sawood14012) [reply .github/workflows/ci-run-e2e.yml:59]: yeah i can add comments

File: .github/workflows/ci-run-e2e.yml (hunk 2 above)
Comment (sawood14012) [reply .github/workflows/ci-run-e2e.yml:37]: yes this is how jenkins runs it but we can change only run the runtest.sh by exporting all vars also should work

File: .github/workflows/ci-build.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.6'
+    - name: Install virtualenv
+      run: pip install -U pip && pip install virtualenv
+    - name: ${{ matrix.scripts }}
+      run: bash qa/${{ matrix.scripts }}.sh;
+
```
Comment (arajkumar) [inline .github/workflows/ci-build.yml:29]: Can you add another job to build docker image, just to sanity check Dockerfile.

**Review by arajkumar — CHANGES_REQUESTED** [review]
few more nits

File: .github/workflows/ci-build.yml (hunk 3 above)
Comment (sawood14012) [reply .github/workflows/ci-build.yml:29]: okay

**Review by arajkumar — APPROVED** [review]
lgtm

**Pull request merged**